tests/network/vmi_networking.go

```diff
@@ -226,7 +229,7 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				inboundVMI = libvmi.NewCirros()
 				inboundVMI, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(inboundVMI)
 				Expect(err).ToNot(HaveOccurred())
-				inboundVMI = tests.WaitUntilVMIReady(inboundVMI, libnet.WithIPv6(console.LoginToCirros))
+				inboundVMI = tests.WaitUntilVMIReady(inboundVMI, console.LoginToCirros)
```

Comment:
@AlonaKaplan Can you please tell how does the fact that a cluster can be IPv6 single stack affects the VMI configuration - in this case that its not needed anymore to configure IPv6 on the VM under test?